test/test_main.cpp

```diff
@@ -197,5 +197,5 @@ QInterfaceTestFixture::QInterfaceTestFixture()
     rng->seed(rngSeed);
 
     qftReg = CreateQuantumInterface(testEngineType, testSubEngineType, testSubSubEngineType, 20, 0, rng,
-        complex(ONE_R1, ZERO_R1), enable_normalization, true, true, device_id, !disable_hardware_rng);
+        complex(ONE_R1, ZERO_R1), enable_normalization, true, false, device_id, !disable_hardware_rng);
```

Comment:
This is why I hate true/false as flag parameters for functions, because it's impossible to tell - at the time of call - what behavior is intended.  Mostly my fault for not keeping the context loaded into my brain, I admit :)

Reply:
I see your point, but we have a convenient Web-based Doxygen, which names these parameters meaningfully. If some IDE autocompletion like Intellisense could pull it from the header, the meaning of each Boolean would be clear from the names in the declaration.

That said, yes, it got me, here. But, rather, what "got" me was the truth/negation convention, correctly given by the name.